static/src/stylesheets/module/email/_form.scss

```diff
@@ -0,0 +1,207 @@
+// TODO These mixins are here while we run the A/B test as the styles are custom to the email form but if they become perm then
+// they should be made reusable
+
+@mixin rounded-input--inline-button() {
+    &text-input {
+        border-right: 0;
+        @include border-radius(1000px 0 0 1000px);
+        float: left;
+        width: 65%;
+    }
+
+    &submit-input {
+        font-size: 12px;
+        @include border-radius(0 1000px 1000px 0);
+        border-left: 0;
+        float: left;
+        margin-right: 0;
+        padding-left: 10px;
+        width: 35%;
+    }
+}
+
+.email-sub {
+    @include clearfix();
+}
+
+.email-sub__iframe {
+    width: 100%;
+}
+
+.email-sub__form {
+    font-family: $f-sans-serif-text;
+    font-size: 14px;
+    padding: 20px;
+
+    * {
+        box-sizing: border-box;
+    }
+
+    &--is-hidden {
```

Comment:
this makes it impossible to grep the code base for these classes - much safer to write out the full class